python/lsst/cell_coadds/_stitched_coadd_inputs.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of cell_coadds.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import logging
+from collections.abc import Iterable, Mapping
+
+import lsst.geom as geom
+from lsst.skymap import Index2D
+
+from ._identifiers import ObservationIdentifiers
+from ._uniform_grid import UniformGrid
+
+__all__ = ("StitchedCoaddInputs",)
+
+logger = logging.getLogger(__name__)
+
+
+class StitchedCoaddInputs:
+    """A class that quacks like BoundedField and defined on a grid piecewise.
```

Comment:
Should be "quacks like ExposureCatalog", I think.  But that might mean the class name is a little off, because `CoaddInputs` is a struct that holds two `ExposureCatalogs`, not an `ExposureCatalog` itself.

Reply:
I always thought the plural in `Inputs` meant that it holds information about multiple visits (and detectors) but are you saying that it is because it holds a `ccd` `ExposureCatalog` and a `visit` `ExposureCatalog`? I only meant this to substitute the `ccd` one since there was no use for `visit` downstream from what I could see.

Reply:
I just wasn't sure if were confusing the `CoaddInputs` class and the thing it holds, since this seemed to be named after the former while acting like the latter.  If you don't need something that acts like the former that's fine.

Reply:
You're right about the confusion. I am modifying this now to have a ``ccds`` attribute at least that is iterable (list of `ObservationIdentifiers`). That will make the changes downstream slightly minimal.

Reply:
I take it back. I'll modify the class (and module) name to reflect it is an `ExposureCatalog`-like object instead of `CoaddInputs`-like object.